ecl2df/common.py

```diff
@@ -621,3 +622,47 @@ def stack_on_colnames(dframe, sep="@", stackcolname="DATE", inplace=True):
     del dframe["level_0"]
     dframe.index.name = ""
     return dframe
+
+def read_zonemap(filename):
```

Comment:
Should we call it `parse_zonemap`,  or even `parse_zonemapfile`?

Reply:
Ok, I agree!